tests/Unit/Serializer/Upcast/UpcastTest.php

```diff
@@ -5,9 +5,10 @@
 namespace Patchlevel\EventSourcing\Tests\Unit\Serializer\Upcast;
 
 use Patchlevel\EventSourcing\Serializer\Upcast\Upcast;
+use PHPUnit\Framework\Attributes\CoversClass;
 use PHPUnit\Framework\TestCase;
 
-/** @covers Patchlevel\EventSourcing\Serializer\Upcast\Upcast */
+#[CoversClass(Patchlevel\EventSourcing\Serializer\Upcast\Upcast::class)]
```

Comment:
We previously had an error in the annotation, which was only now noticed through the attributes:

```suggestion
#[CoversClass(Upcast::class)]
```

Reply:
Good catch ! Thanks for your review.